src/SFA.DAS.LevyTransferMatching.Web/Validators/Pledges/ApplicationsPostRequestValidator.cs

```diff
@@ -0,0 +1,13 @@
+﻿using FluentValidation;
+using SFA.DAS.LevyTransferMatching.Web.Models.Pledges;
+
+namespace SFA.DAS.LevyTransferMatching.Web.Validators.Pledges
+{  
+    public class ApplicationsPostRequestValidator : AbstractValidator<ApplicationsPostRequest>
+    {
+        public ApplicationsPostRequestValidator()
+        {
+            RuleFor(x => x.ApplicationsToReject).NotNull().WithMessage("You must choose applications from the list to reject");
```

Comment:
This message doesn't surface in the UI

Reply:
It only shows a message "There is a problem" but we want to show the actual message which is linked with the validator rule. And I cannot able to figure out why it's not displaying the message which is defined in the validator "Rule". Will need some help with this.

Reply:
The error messages are now working for these validators.